### DISCUSSION TITLE
Manual positioning instead of reference object

### DISCUSSION THREAD

**Discussion by michael** [floating-ui/floating-ui#1208] [Q&A] [answered]
Hi there!

In v1 there was a way to provide a custom referenceObject instead of a DOM Element to achieve manual positioning. 

See: https://popper.js.org/docs/v1/#referenceobject

Can I still do this in Popper v2 and react-popper (with hooks)? I didn't find anything in the docs.

Thanks a lot, great library btw!

**Comment by atomiks** [comment, marked as answer]
https://popper.js.org/docs/v2/virtual-elements/

**Comment by michael** [comment]
Ah sorry, that I overlooked. Thanks a lot!
